### DISCUSSION TITLE
Branching Strategies for Version 3.x

### DISCUSSION THREAD

**Discussion by ydah** [rubocop-hq/rubocop-rspec#1857] [General]
> Should we create a `2-x-stable` branch to backport non-v3 changes to while we prepare for the upcoming v3 release?
> https://github.com/rubocop/rubocop-rspec/pull/1833#issuecomment-2028622466

**Comment by bquorning** [comment]
Yes

**Comment by bquorning** [comment]
I have created a `2-x-stable` branch, and added identical branch protection rules that we have on `master`. The `master` branch is now used for preparing rubocop-rspec v3.

**Reply by bquorning** [reply]
cc @pirj

**Comment by bquorning** [comment]
Closing this discussion in favor of #1880.
